bc_obps/registration/tests/test_models.py

```diff
@@ -802,6 +813,57 @@ def test_unique_swrs_facility_id_constraint(self):
             invalid_operation.save()
 
 
+class FacilityModelTest(BaseTestCase):
+    fixtures = [
+        ADDRESS_FIXTURE,
+        USER_FIXTURE,
+        CONTACT_FIXTURE,
+        OPERATOR_FIXTURE,
+        OPERATION_FIXTURE,
+        DOCUMENT_FIXTURE,
+        BC_OBPS_REGULATED_OPERATION_FIXTURE,
+        FACILITY_FIXTURE,
+    ]
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.test_object = Facility.objects.first()
+        cls.field_data = [
+            *timestamp_common_fields,
+            ("id", "ID", None, None),
+            ("name", "name", 1000, None),
+            ("type", "type", 1000, None),
+            ("operation", "operation", None, None),
+            ("address", "address", None, None),
+            ("new_entrant", "new entrant", None, None),
+            ("well_authorization_numbers", "well authorization number", None, None),
```

Comment:
@Sepehr-Sobhani , I've got these tests working, but I don't think I completely understand how they work. `well_authorization_numbers` is my related name in the Facility table:
```
facility = models.ForeignKey(Facility, on_delete=models.DO_NOTHING, related_name="well_authorization_numbers")
```
But what is `well authorization number`?

Reply:
That is the label name of that field. It's being mainly used for Django forms, but we thought it would be handy to test it out from the very beginning in case we find a use for it later on.